src/Sylius/Behat/Page/Shipping/ShippingMethodDetailsPage.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Shipping;
+
+use Sylius\Behat\Page\ElementNotFoundException;
+use Sylius\Behat\Page\SymfonyPage;
+
+/**
+ * @author Jan Góralski <jan.goralski@lakion.com>
+ */
+class ShippingMethodDetailsPage extends SymfonyPage
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getRouteName()
+    {
+        return 'sylius_backend_shipping_method_show';
+    }
+
+    public function pressDelete()
```

Comment:
If we want to be as explicit as it is possible, you should add docblock with throws statement